src/styles/styles.js

```diff
@@ -324,6 +328,16 @@ const styles = {
         backgroundColor: themeColors.buttonDefaultBG,
     },
 
+    buttonMedium: {
+        borderRadius: variables.componentBorderRadiusNormal,
+        height: variables.componentSizeNormal,
+        paddingTop: 6,
+        paddingRight: 12,
+        paddingBottom: 6,
+        paddingLeft: 12,
+        backgroundColor: themeColors.buttonDefaultBG,
```

Comment:
It's weird that `buttonSmall` and `buttonMedium` both provide a default background color, while `buttonLarge` does not. IMO the `backgroundColor` could be a different style or should be included in `buttonLarge`.

Reply:
Hmm, I copied the styles from small and then just updated the padding. Hence the issue. I can create add the backgroundColor to `buttonLarge`.